### ISSUE TITLE
Grants description JSON, not plain text

### ISSUE THREAD

**Issue by NukeManDan** [gitcoinco/web#5748]
**Describe the bug**
Two issues:
- [ ] I cannot edit my grant description
- [ ] It is (as pictured) not rendering the text outside a JSON object.

https://gitcoin.co/grants/220/cryptorado/

**To Reproduce**
Any browser I try.

OS: Ubuntu MATE, Desktop
Browser: Firefox, Brave. 

**Screenshots**

![image](https://user-images.githubusercontent.com/35669742/72026200-eb455100-3237-11ea-8972-540c91517b0a.png)

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 0.4 ETH (54.43 USD @ $136.07/ETH)  attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/5748/3875).
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/gitcoinco/web/5748/3875).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $73,537.09 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 1 month from now.
Please review their action plans below:


**1) [iamonuwa](https://gitcoin.co/iamonuwa) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/gitcoinco/web/5748/3875?mutate_worker_action=approve&worker=iamonuwa) | [reject worker](https://gitcoin.co/issue/gitcoinco/web/5748/3875?mutate_worker_action=reject&worker=iamonuwa))_.**

Here to fix this as well. Investigate and get tit resolved.

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/5748/3875).

**Comment by celebritydeveloper** [comment]
Please, a quick question. Where do I get the codebase from?

**Comment by NukeManDan** [comment]
@celebritydeveloper - what do you mean....? the codebase for the website?

**Comment by celebritydeveloper** [comment]
Yes, I mean codebase for the website.

**Comment by celebritydeveloper** [comment]
> @celebritydeveloper - what do you mean....? the codebase for the website?

Yes, I mean the codebase of the website.

**Comment by owocki** [comment]
this seems ready https://github.com/gitcoinco/web/issues/5754

**Comment by owocki** [comment]
@celebritydeveloper codebase is https://github.com/gitcoinco/web

**Comment by TomAFrench** [comment]
It seems to work fine for me on those platforms. Just to make sure... are you blocking javascript at all in firefox and brave?

**Comment by TomAFrench** [comment]
It seems to me that this issue is completely consistent with javascript being blocked in your browser (both the json display and lack of ability to edit) as shown in this code snippet

https://github.com/gitcoinco/web/blob/fe97b965ae4a5d25f34be59c328cea5299aadaf3/app/grants/templates/grants/detail/tabs.html#L135-L158

**Comment by thelostone-mc** [comment]
Thanks for the report @NukeManDan !
Got a fix ready !

@TomAFrench Ah yes sir! you've arrived at the root of the problem :D 
I wish I'd seen this before starting to fix it :P would have saved me some time BUT THANKSS ❤️

**Comment by thelostone-mc** [comment]
This is fixed and deployed ^_^
Closing this out

**Comment by NukeManDan** [comment]
This is diplaying correcly now, but I am unable to edit in Firefox.

All javascript unblocked, and still unable to click any button. https://gitcoin.co/grants/220/cryptorado?tab=description

![image](https://user-images.githubusercontent.com/35669742/72180243-f7e6b800-33a3-11ea-808a-18ff5fbbaca7.png)


I get an error:
```
eferenceError: grant_description is not defineddetail.cfd55a0bd85d.js:41:9
    <anonymous> detail.cfd55a0bd85d.js:41
    jQuery 2
    d raven.js:445
```

Raven.js is running in my browser as well.

**Comment by NukeManDan** [comment]
@thelostone-mc - mid reopening this until a fix is made?

**Comment by owocki** [comment]
i can reproduce here too https://bits.owocki.com/jkunX8Pp

**Comment by owocki** [comment]
since @thelostone-mc went to bed (hes in india), i just put in a fix. mind testing now?

**Comment by NukeManDan** [comment]
Fixed. 
Minor UX request - edit button unlocks ability to edit deception but the editor is enabled without hitting this. You can't edit until you hit the button, but it _feels_ like you should be able to.

New issue for the backlog? Or just me?

**Comment by thelostone-mc** [comment]
Ah thanks @owocki + @NukeManDan !
I should have caught that 🤦‍♂ 

@NukeManDan I'm closing this out now. Do reopen if anything related to this happens again

> Minor UX request - edit button unlocks ability to edit deception but the editor is enabled without hitting this. You can't edit until you hit the button, but it feels like you should be able to.

Just pushed this in ! :)

**Comment by gitcoinbot** [comment]
@celebritydeveloper Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/5748/3875?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/5748/3875?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/5748/3875?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/5748/3875?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/5748/3875?snooze=100)

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Cancelled** 

<hr>

__The funding of 0.4 ETH  (57.34 USD @ $143.34/ETH) attached to this issue has been **cancelled** by the bounty submitter__

 
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $80,263.80 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)
